app/scripts/controllers/persistentVolumeClaim.js

```diff
@@ -54,6 +57,15 @@ angular.module('openshiftConsole')
       DataService
         .get($scope.pvcVersion, $routeParams.pvc, context, { errorNotification: false })
         .then(function(pvc) {
+          var storageClassName = storageClass(pvc);
+          if(storageClassName) {
+            DataService.get($scope.storageClassesVersion, storageClassName, {namespace: pvc.metadata.namespace}).then(function(storageClassData) {
```

Comment:
Storage classes are cluster scoped, correct? You should be able to leave off namespace here

Reply:
@spadgett I don't think it's cluster scoped.

I removed namespace and it is also giving an error.

Reply:
What is the error?

Reply:
Ahh! ok I wasn't passing {}.

Reply:
now it worked, thanks!